massdriver-application-azure-app-service/dns.tf

```diff
@@ -1,7 +1,7 @@
 locals {
   zone_split_id       = var.dns.enable_dns ? split("/", var.dns.zone_id) : []
-  zone_name           = element(local.zone_split_id, index(local.zone_split_id, "dnszones") + 1)
-  zone_resource_group = element(local.zone_split_id, index(local.zone_split_id, "resourceGroups") + 1)
+  zone_name           = var.dns.enable_dns ? element(local.zone_split_id, index(local.zone_split_id, "dnszones") + 1) : null
```

Comment:
This change is because in Terraform you can't call `index` on an empty array. Okie dokie.